editor/src/components/canvas/ui-jsx-canvas-renderer/scene-root.tsx

```diff
@@ -197,7 +248,7 @@ export const SceneRootRenderer = betterReactMemo(
             mutableUtopiaContext.jsxFactoryFunctionName,
             sceneProps.component,
             propsWithScenePath,
-            undefined,
+            renderedChildren,
```

Comment:
In hindsight there was absolutely no point to adding the `renderedChildren` as children of the `rootElement` here. I'm immediately changing that in the next piece of work (that I've started), and instead in the return statement below, I'm replacing line 271:
```
{rootElement}
```
with:
```
<>
  {rootElement}
  {renderedChildren}
</>
```
I can't add that comment on that line because it's not included in the git diff's context.
That change will come with the next PR later today.